### DISCUSSION TITLE
Calculating RA/DEC value in Stellarium from Hipparcos Catalog

### DISCUSSION THREAD

**Discussion by tsuiwade** [Stellarium/stellarium#2892] [Q&A] [answered]
I have no background in astronomy. I have been wanting to write code to make star charts for a given location and time, which led to this question.

I've downloaded the Hipparque Catalog. I want to calculate the same RA/DEC values as those in the Stellarium from the Hipparcos Catalog. What should I do?

I realize that Hipparcos is the value obtained from J1991.25. I adjusted the time of Stellarium to J1991.25(13:30:0.00 UT on April 2, 1991). and J2000(12:00:0.00 UT on January 1, 2000), but the RA/DEC value displayed by Stellarium is different from the value in Hipparcos Catalog. Is there anything else to add?

So how to calculate J2000 RA/DEC at different times in Stellarium? I realize that its value changes with time?

**Comment by gzotti** [comment, marked as answer]
Please do not use Stellarium as source or reference for own work. It has its own errors. 

That said, to compare, set Stellarium to J1991.25 (to set proper motion), and observe the J2000 coordinates. At least disable aberration and probably even nutation. Or observe from the center of the Sun.
